### PR TITLE
#3 Header & What We Do

### LINKED ISSUE #3: Header & Main Claim

### LINKED ISSUE #4: What We Do

### PR DESCRIPTION
**Pull request by tmrdlt**
## Description
- Add Main Claim and sticky Header component with fade in effect
- Add scroll down button
- Add What We Do

## Screenshots
Desktop
grafik
grafik
Mobile

### CODE REVIEW

File: src/app/page.tsx [hunk 1]
```diff
@@ -14,12 +17,11 @@ import * as React from 'react';
 export default function HomePage() {
   return (
     <main>
-      <Head>
-        <title>Hi there</title>
-      </Head>
-      <div className='bg-primary-cyan h-20 w-full font-mono'>Unit 214</div>
-      <div className='bg-primary-red h-20 w-full'>Coming</div>
-      <div className='bg-primary-amber h-20 w-full'>soon...</div>
+      <div className='h-[4000px]'>
```
Comment (kenodressel) [inline src/app/page.tsx:20]: why the fixed height?

File: src/components/MainClaim.tsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+
+export default function MainClaim() {
+  return (
+    <div className='text-main-mobile lg:text-main px-10 font-mono font-extrabold text-white lg:px-28'>
+      <a className='text-white/50'> UNLOCK YOUR </a> POTENTIAL
```
Comment (kenodressel) [inline src/components/MainClaim.tsx]: why `a` as a tag?

File: src/app/page.tsx (hunk 1 above)
Comment (tmrdlt) [reply src/app/page.tsx:20]: This was only for testing the scrolling. I will remove it later.

File: src/components/MainClaim.tsx (hunk 2 above)
Comment (tmrdlt) [reply src/components/MainClaim.tsx]: Well, good question 😆 I think I wanted to use` <p> `here. But probably `<span>` makes more sense in this case. I'll change it in all places 👍

**Comment by kenodressel** [comment]
I am getting a weird aliasing on the edge of the box:
System: Chrome Version 119.0.6045.123 (Official Build) (64-bit)
OS: Arch Linux

image

**Review by kenodressel — APPROVED** [review]
issue moved to #19

**Pull request merged**